resource-managers/yarn/src/test/resources/META-INF/services/org.apache.spark.deploy.yarn.security.ServiceCredentialProvider

```diff
@@ -1 +0,0 @@
-org.apache.spark.deploy.yarn.security.TestCredentialProvider
```

Comment:
@vanzin Do you know the purpose of this test service provider?  I don't see this class anywhere in the code base.